cranelift/filetests/filetests/egraph/algebraic.clif

```diff
@@ -379,8 +378,9 @@ block0(v0: i64):
     v3 = iconst.i64 3
     v4 = bor v3, v2
     return v4
-    ; check: v6 = bor v0, v3
-    ; check: return v6
+    ; check: v3 = iconst.i64 3
```

Comment:
We don't bother checking values which were in the input for these tests, so let's remove this `check: v3` line.